packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -17,18 +17,13 @@ import { Link } from '../../lib/reactRouterWrapper';
 import { AnalyticsContext, useTracking } from '../../lib/analyticsEvents'
 import seedrandom from '../../lib/seedrandom';
 
-const YEAR = 2019
-const NOMINATIONS_VIEW = "nominations2019"
-const VOTING_VIEW = "voting2019" // unfortunately this can't just inhereit from YEAR. It needs to exactly match a view-type so that the type-check of the view can pass.
-const REVIEW_COMMENTS_VIEW = "reviews2019"
-const userVotesAreQuadraticField: keyof DbUser = "reviewVotesQuadratic2019";
+export const REVIEW_YEAR = 2020
+const VOTING_VIEW = "roughVoting2020" // unfortunately this can't just inhereit from REVIEW_YEAR. It needs to exactly match a view-type so that the type-check of the view can pass.
```

Comment:
There are ways you could hack this (type assertion, or looking the view up in an object), but I don't think they're worth it.